vmdb/db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Provisioning/StateMachines/Methods.class/__methods__/postprovision.rb

```diff
@@ -0,0 +1,14 @@
+#
+# Description: This method examines the orchestration stack provisioned
+#
+$evm.log("info", "Starting Orchestration Post-Provisioning")
+
+# In the following example all the outputs from the stack are dumped.
+# You can add logic to process the stack object in VMDB
+service = $evm.root["service_template_provision_task"].destination
+stack = $evm.vmdb('orchestration_stack').find_by_ems_ref(service.stack_ems_ref)
```

Comment:
We should be able to ask the service for its stack instead of having to load it this way.  There are a couple of options here so let's discuss.